extensions/pom.xml

```diff
@@ -110,18 +109,6 @@
       <artifactId>commons-fileupload</artifactId>
       <scope>compile</scope>
     </dependency>
-    <dependency>
-      <groupId>org.apache.httpcomponents</groupId>
-      <artifactId>httpclient</artifactId>
-      <version>${httpclient.version}</version>
```

Comment:
Did you double check if the httpclient lib was still being included in the `lib`? It probably isn't it is probably needed...

Reply:
A pipeline was tested without this, and the lib was still being included.